pkgs/tools/admin/docker-credential-helpers/default.nix

```diff
@@ -44,6 +44,7 @@ buildGoPackage rec {
     homepage = "https://github.com/docker/docker-credential-helpers";
     license = licenses.mit;
     maintainers = [ maintainers.marsam ];
+    mainProgram = "docker-credential-osxkeychain";
```

Comment:
not sure that's the one to choose

Reply:
Hmm, I guess that's the only executable that shows up since I'm on macOS. I'll just remove this from the PR. Good catch!